File: CPP/SentenceSmash/MySolution.cpp
#include <vector>
#include <string>
#include <numeric>

std::string smash(const std::vector<std::string>& words)
{
    return std::accumulate(words.begin(), words.end(), std::string{},
        [](const std::string& a, const std::string& b) {
            return a.empty() ? b : a + ' ' + b;
        });
}

// Original kata: https://www.codewars.com/kata/53dc23c68a0c93699800041d
// My solution: https://www.codewars.com/kata/reviews/6632c2c62f94ad0001da4294/groups/66bbd821ab5125f05d8a4994

File: PHP/Deadfish/MySolution.php
<?php

function parse($data) {
  $result = 0;
  $array = [];
  foreach (str_split($data) as $g) {
    if ($g == 'i'){
      $result++;
    }
    else if($g == 'd'){
      $result--;
    }else if($g == 's'){
      $result = $result * $result;
    }else if($g == 'o'){
      array_push($array, $result);
    }
  }
  return $array;
}

// Original kata: https://www.codewars.com/kata/51e0007c1f9378fa810002a9
// My solution: https://www.codewars.com/kata/reviews/5b8e53645a5f104129000811/groups/67eb7487a6458b2a6d958033
